### PR TITLE
Overwrite Event organization for single-org users if no org is passed along by request

### PR DESCRIPTION
**Pull request by susannasiebert**
The frontend should submit the organization of a user to attribute an Event to but it won't if someone runs an outdated version. This will at least catch cases for single-org users.

### CODE REVIEW

File: app/models/actions/propose_assertion.rb [hunk 1]
```diff
@@ -26,16 +27,16 @@ def execute
...
-        organization: organization
-      )
+      @subject = assertion
+      create_event('assertion submitted')
       assertion.subscribe_user(originating_user)
       @assertion = assertion
     end
 
+    def state_params
+      nil
```
Comment (acoffman) [inline app/models/actions/propose_assertion.rb:37]: Do assertions not have state params? We do link to them from the events right?

File: app/models/actions/propose_evidence_item.rb [hunk 2]
```diff
@@ -22,17 +23,17 @@ def execute
...
-      )
+      @subject = evidence_item
+      create_event('submitted')
       evidence_item.subscribe_user(originating_user)
       process_source_suggestion
       @evidence_item = evidence_item
     end
 
+    def state_params
+      nil
```
Comment (acoffman) [inline app/models/actions/propose_evidence_item.rb:34]: Same question here

File: app/models/actions/suggest_publication.rb [hunk 3]
```diff
@@ -80,13 +82,8 @@ def create_publication_submission
...
 
-    def create_event
-      Event.create(
-        action: 'publication suggested',
-        originating_user: originating_user,
-        subject: source,
-        organization: organization
-      )
+    def state_params
+      nil
```
Comment (acoffman) [inline app/models/actions/suggest_publication.rb:86]: Ditto

File: app/models/actions/propose_assertion.rb (hunk 1 above)
Comment (susannasiebert) [reply app/models/actions/propose_assertion.rb:37]: The deleted portion that created the Event previously wasn't setting the state params. I'm confused what differentiates the Event's state params from the subject's state params. The subject/assertion themselves does have state params.

**Pull request merged**